src/cogs/admin_commands.py

```diff
@@ -33,11 +33,11 @@ async def shutdown(self, ctx):
 
     @shutdown.error
     async def shutdown_error(self, ctx, error):
-        await ctx.channel.send("you're not an owner :cry:")
+        await ctx.channel.send(f"I'm sorry, {ctx.author}, I'm afraid I can't do that for you. You're not an owner :cry:")
 
     async def cog_command_error(self, ctx, error):
         if isinstance(error, commands.MissingPermissions):
-            await ctx.send("You can't do that")
+            await ctx.send(f"I'm sorry, {ctx.author}, I'm afraid I can't do that for you.")
```

Comment:
will this work correctly with the mention?

Reply:
To have it mentioned it should be `ctx.author.mention`